src/soundfile.js

```diff
@@ -886,6 +886,14 @@ class SoundFile {
   }
 
   // TO DO: document this
+  /**
+   *  Set the pitch rate of a sound file. Will change the pitch and the speed also.
+   *
+   *  @method setPitch
+   *  @for p5.SoundFile
+   *  @param {Number} pitchRate     Set the pitch rate.
```

Comment:
OK, Then I think we should write like this :
Line 894: Set the pitch by passing the MIDI node value. If increase the value of MIDI then the frequency will increase as a result it will also affect the rate and other factors of the sound file.

Reply:
That's getting closer for sure! I think we can be a little more specific: instead of mentioning "other factors," we can say "If the MIDI note is increased, then both the frequency of the sound and its playback speed will increase as a result."